test/extended/cpu_partitioning/crio.go

```diff
@@ -306,10 +306,13 @@ func collectContainerInfo(ctx context.Context, oc *exutil.CLI, node corev1.Node)
 	info := []crioContainerData{}
 	err = retry.OnError(backoff, shouldRetryExec,
 		func() error {
-			out, _, execErr := e2epod.ExecWithOptions(oc.KubeFramework(), execOptions)
+			out, outStdErr, execErr := e2epod.ExecWithOptions(oc.KubeFramework(), execOptions)
 			if execErr != nil {
 				return execErr
 			}
+			if outStdErr != "" {
+				return fmt.Errorf("err execing command %s", outStdErr)
+			}
```

Comment:
I'm still unconvinced that having -any- data on stderr means we failed something. At very least this should be bound to the specific command we run (and still we're assuming this never changes) with a code comment.

Reply:
I might me misunderstanding then. In our scenario here we're executing this script in the container. Would it not be only errors that come through the stderr in this situation?